tests/runtime/builtin

```diff
@@ -193,3 +193,8 @@ PROG BEGIN { printf("size=%d\n", sizeof(struct task_struct)); exit(); }
 EXPECT ^size=
 TIMEOUT 5
 REQUIRES_FEATURE btf
+
+NAME offsetof
+PROG struct Foo { int x; long l; char c; } BEGIN { printf("%ld %ld\n", offsetof(struct Foo, c), offsetof(struct Foo, c)); }
+EXPECT ^0 16$
```

Comment:
Same as with the codegen test, the value '16' depends on ABI, so I'd stick with checking the offset of `x`.